src/cmds/query.ts

```diff
@@ -36,6 +41,7 @@ export async function handler(
 
   const endpoint = config.endpointsExtension.getEndpoint(argv.endpoint)
   const query = fs.readFileSync(argv.file, { encoding: 'utf8' })
+  const variables = argv.variables || "{}"
```

Comment:
There is a ts-lint error on this line in the CI build from the use of double quotes.